core/src/main/antlr4/SelectLexer.g4

```diff
@@ -0,0 +1,37 @@
+
+lexer grammar SelectLexer;
+
+@header {
+package se.tre.freki.query;
+}
+
```

Comment:
Would be nice with comments in this file describing the different fields for future use